network/NetworkGen/classes/Can.py

```diff
@@ -394,6 +408,13 @@ def __init__(self, node: "CanNode", name: str, msg_def: dict):
         self.node_ref = None
         self.is_valid = False
 
+    def crc32(self):
+        crc = crc32(self.name.encode())
```

Comment:
Crc'ing the name of the message is maybe not super relevant. All of software piggybacks off the signal name so this might be overly restrictive for nothing

Reply:
removed name and expressly added validator signals as additional hash